### ISSUE TITLE
Content pages: content filters are after the publications list in the tab order

### ISSUE THREAD

**Issue by eatyourgreens** [zooniverse/front-end-monorepo#1699]
**Package**
app-content-pages

**Describe the bug**
For a filtered list to be useful, the filters should be placed before the list both in the tab order and the page reading order.
Tabbing through the publications list, the filter controls are at the end of the page. You have to tab through the entire list before you can filter it.
https://www.zooniverse.org/about/publications

The bug is in `TwoColumnLayout` here, where the sidebar is placed after the main content.
https://github.com/zooniverse/front-end-monorepo/blob/8309c0ca347eca3b875be1c460ca164f122c3639/packages/app-content-pages/src/shared/components/TwoColumnLayout/TwoColumnLayout.js#L13-L28

**Comment by tanayv** [comment]
Hello! Is this still an issue and can I take it up? Thanks!

**Comment by srallen** [comment]
@tanayv yes it is and you're welcome to take it! Have a look at our readme on getting started with our repo: https://github.com/zooniverse/front-end-monorepo/blob/master/README.md and do let us know if you have questions.
